src/navigation/lightning/LightningNavigator.tsx

```diff
@@ -30,7 +30,9 @@ export type LightningStackParamList = {
 		orderId: string;
 	};
 	Result: undefined;
-	QuickSetup: undefined;
+	QuickSetup: {
+		headerTitle: string;
+	};
```

Comment:
To fix the type error:
```suggestion
	QuickSetup: {
		headerTitle: string;
	} | undefined;
```